var/spack/repos/builtin/packages/py-fenics-ufl-legacy/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyFenicsUflLegacy(PythonPackage):
+    """The Unified Form Language (UFL) is a domain specific language for
+    declaration of finite element discretizations of variational forms. More
+    precisely, it defines a flexible interface for choosing finite element
+    spaces and defining expressions for weak forms in a notation close to
+    mathematical notation."""
+
+    homepage = "https://fenicsproject.org/"
+    url = "https://github.com/FEniCS/ufl-legacy/archive/2022.3.0.tar.gz"
```

Comment:
https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi-vs-github

Same for your other packages, but if you want to keep them downloading from GitHub for some reason, that's also fine.

Reply:
I see, I think in other to be consistent with the other FEniCS packages it is best to just keep it as is.